CONTRIBUTING.md

```diff
@@ -0,0 +1,100 @@
+<!--
+SPDX-FileCopyrightText: 2022 Contributors to the Power Grid Model IO project <dynamic.grid.calculation@alliander.com>
+SPDX-License-Identifier: MPL-2.0
+-->
+
+# How to Contribute
+
+We'd love to accept contributions to this project in any forms. 
+You do not need to be a programming or power system expert to make a contribution.
+There are just a few small guidelines you need to follow before making a change.
+
+## Ways of contributing
+
+Contribution does not necessarily mean committing code to the repository. 
+We recognize different levels of contributions as shown below in increasing order of dedication:
+
+1. Test and use the workshop. Give feedback on the user experience or suggest new features.
+2. Report bugs.
+3. Improve the workshop
+
```

Comment:
The ways of contribution will be slightly different here. Testing, using, new features, bugs, improvements will be present. Also,

- Add more conversions from other power system analysis libraries to power-grid-model

Reply:
Good feedback. I will updat this.